internal/pkg/remote/endpoint/client_test.go

```diff
@@ -136,74 +134,68 @@ func TestKeyserverClientOpts(t *testing.T) {
 }
 
 //nolint:dupl
-func TestRegistryClientConfig(t *testing.T) {
+func TestLibraryClientConfig(t *testing.T) {
```

Comment:
@DrDaveD @gmkurtzer not sure all of our unit tests should depend on 3rd party commercial server -- this also regresses our newer GHCR server for containers

Reply:
A remote endpoint is really almost all about the library interface, with the rest little bit for key servers.  (It used to also be about build services, but that's removed).  It doesn't do anything related to a general registry service.  Those are always accessed directly by URL, they are not registered endpoints.